### PR TITLE
Design changes #156

### PR DESCRIPTION
**Pull request by fbbunny**
- changed navigation bar so the subpage you're on is highlighted in the navigation bar
- changed colors
- changed fonts 
- changed frontpage pictures (Which I still agree, we need to change - any suggestions?) 

Suggestion for the following welcome: can't figure out why is the Cat in TrackCat grey, if it's not defined so in css, I was also unsucessful in making the TrackCat sing on the front page responsive in terms of size

### CODE REVIEW

**Comment by gandalfar** [comment]
It's in grey because of this:
```strong {
	color: #5C5C5C;
}
```

**Comment by gandalfar** [comment]
#frontpage should be `position: relative`
#frontpage_title change it to: `position: absolute; bottom: 0; left: 0;` and remove margins and paddings and it will behave a bit better

**Comment by ialja** [comment]
Ah, @gandalfar beat me to it :grin: Overall, visually a nice change :+1: Perhaps just a little less empty space above the content in the navbar?

File: static/css/style.css [hunk 1]
```diff
@@ -115,13 +122,24 @@ code {
...
 }
 
 h1 {
-	text-align: center;
 	position: relative;
+	margin: 105px 0px 15px 0px;
+}
+
+#about {
+text-align: center;
```
Comment (ialja) [inline static/css/style.css]: Just a lil' indent missing :)

**Comment by ialja** [comment]
Also, please check what's going on with the project_detail page - the title is completely covered by the navbar.

screen shot 2015-02-23 at 16 12 07

**Comment by ialja** [comment]
This PR made me happy, thank yew! :ok_woman: :+1:

**Pull request merged**